xml/cap_depl_stratos.xml

```diff
@@ -145,4 +145,253 @@ NAME                    CLUSTER-IP      EXTERNAL-IP    PORT(S)         AGE
+      <listitem>
+	<para>For CaaSP, use the Auth Type <literal>CAASP (OIDC)</literal> and provide a valid
+	<filename>kubeconfig</filename> file for your environment</para>
+      </listitem>
+
+      <listitem>
+	<para>For Amazon EKS, use the Auth Type <literal>AWS IAM (EKS)</literal> and provide the
+        name of your EKS cluster and your AWS Access Key ID and Secret Access
+        Key</para>
+      </listitem>
+
+      <listitem>
+        <para>For Azure AKS, use the Auth Type <literal>Azure AKS</literal> and provide a valid
+	<filename>kubeconfig</filename> file for your environment</para>
+      </listitem>
+
+      <listitem>
+        <para>For Minikube, use the Auth Type <literal>Kubernetes Cert Auth</literal> and
+        provide the Certificate and Certificate Key files</para>
+      </listitem>
+    </itemizedlist>
+
+    <para>Finally, click <guimenu>Connect</guimenu> to connect the endpoint with the
+    authentication information that you have provided. The endpoint list
+    should update to show that your Kubernetes endpoint is connected.</para>
+
+    <para>Once connected, you should see a Kubernetes menu item in the
+    left-hand side navigation - click on this to access Kubernetes
+    views.</para>
+  </sect1>
+
+  <sect1 xml:id="sec.cap.stratos-metrics">
+   <title>Stratos Metrics</title>
+
+   <para>Stratos can show metrics data from
+    <link xlink:href="https://prometheus.io/">Prometheus</link> for both Cloud Foundry
+    and Kubernetes.</para>
+
+   <sect2 xml:id="sec.cap.stratos-metrics-install">
+    <title>Install Stratos Metrics with &helm;</title>
+
+    <para>In order to display metrics data with Stratos, you need to deploy the
+    <literal>stratos-metrics</literal> &helm; chart - this deploys Prometheus with
+    the necessary exporters that collect data from Cloud Foundry and Kubernetes.
+    It also wraps Prometheus with an <link xlink:href="http://nginx.org/">nginx</link>
+    server to provide authentication.</para>
+
+    <para>As with deploying Stratos, you should deploy the metrics &helm; chart
+    using the same <filename>scf-config-values.yaml</filename> file that was used for
+    deploying SCF and UAA.</para>
+
+    <para>Create a new yaml file named <filename>stratos-metrics-values.yaml</filename>,
+    with the following contents:</para>
+
+<screen>kubernetes:
+  authEndpoint: <replaceable>$KUBE_SERVER_ADDRESS</replaceable>
+prometheus:
+  kubeStateMetrics:
+    enabled: true
+nginx:
+  username: <replaceable>$USERNAME</replaceable>
+  password: <replaceable>$PASSWORD</replaceable></screen>
+
+    <para>where:</para>
+
+    <itemizedlist>
+      <listitem>
+        <para><replaceable>$KUBE_SERVER_ADDRESS</replaceable> is the same URL that you
+        used when registering your Kubernetes environment with Stratos (the Kubernetes
+        API Server URL)</para>
+      </listitem>
+
+      <listitem>
+        <para><replaceable>$USERNAME</replaceable> should be chosen by you as the
```

Comment:
We use a lot of environment variables in this guide, e.g. $ENVVAR. I suggest not using the dollar sign for replaceable elements, as that could be confusing.